samples/MonoGame/README.md

```diff
@@ -0,0 +1,90 @@
+> dotnet new mgdesktopgl -o MyGame
+> cd MyGame
+```
+
+This will install .NET Core MonoGame template and create empty game project. .NET Core MonoGame port lives at https://github.com/cra0zy/MonoGame/tree/core currently. Thank you @cra0zy for the great work! 
+
+Verify that the empty game runs. You should see light blue window:
+
+```bash
+> dotnet run
+```
+
+## Add CoreRT to your project
+Using CoreRT to compile your application is done via the ILCompiler NuGet package, which is [published to MyGet with the CoreRT daily builds](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.DotNet.ILCompiler).
+For the compiler to work, it first needs to be added to your project.
+
+In your shell/command prompt navigate to the root directory of your project and run the command:
+
+```bash
+> dotnet new nuget 
+```
+
+This will add a nuget.config file to your application. Open the file and in the ``<packageSources> `` element under ``<clear/>`` add the following:
+
+```xml
+<add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+```
+
+Once you've added the package source, add a reference to the compiler by running the following command:
+
+```bash
+> dotnet add package Microsoft.DotNet.ILCompiler -v 1.0.0-alpha-* 
+```
+
+## Restore and Publish your app
+
+Once the package has been successfully added it's time to compile and publish your app! If you're using Windows, make sure you're using `x64 Native Tools Command Prompt for VS 2017` instead of the standard Windows command prompt. In the shell/command prompt window, run the following command:
+
+```bash
+> dotnet publish -r <RID> -c <Configuration>
+```
+
+where `<Configuration>` is your project configuration (such as Debug or Release) and `<RID>` is the runtime identifier, which you specified in the csproj file (one of win-x64, linux-x64, osx-x64). For example, if you want to publish a release configuration of your app for a 64-bit version of Windows the command would look like:
+
+```bash 
+> dotnet publish -r win-x64 -c release
+```
+
+Once completed, you can find the native executable in the root folder of your project under `/bin/x64/<Configuration>/netcoreapp2.0/publish/`. Navigate to `/bin/x64/<Configuration>/netcoreapp2.0/publish/` in your project folder and run the produced native executable.
+
+## Try MonoGame sample game
+
+Clone MonoGame samples from github:
+
+```bash
+> git clone https://github.com/MonoGame/MonoGame.Samples
+```
+
+MonoGame samples include project files for number of targets, but not for .NET Core yet. One has to create the project using above steps and transplant links to sample sources and assets to it. This directory contains .NET Core project for NeonShooter sample that assumes MonoGame.Samples is cloned under it. Build it and start shooting!
+
+```bash
+> dotnet publish -r win-x64 -c release NeonShooter.csproj
```

Comment:
@MichalStrehovsky This step fails with ` VTable of type 'Microsoft.Xna.Framework.Content.ContentTypeReader`1<System.Collections.Generic.List`1<System.__Canon>>' not computed by the IL scanner.` error. It is recent regression. Could you please take a look?

Reply:
#5228 fixes it. Sorry for that!